src/app/components/sidebar/nav-data.ts

```diff
@@ -0,0 +1,25 @@
+
+export const navbarData=[
+    
+    {
+        routerLink:'/profil',
+        icon :'fal fa-user-circle',
+        label:'Profil'
+    },
+    {
+        routerLink :'/taches',
+        icon :'fal fa-tasks',
+        label:'Tâches'
+    },
+    {
+        routerLink:'message-page',
+        icon :'fal fa-envelope',
+        label:'Messages'
+    },
+    {
+        routerLink:'/profil',
+        icon :'fal fa-users',
+        label:'Groupes'
+    }
+
+];
```

Comment:
the name navbarData can be confused with the main navbar. change the name "navbarData" to "sidebarData"